sixel/__init__.py

```diff
@@ -54,10 +54,23 @@ def _filenize(f):
 
     mode = os.fstat(f.fileno()).st_mode
     if stat.S_ISFIFO(mode) or os.isatty(f.fileno()):
+        have_stringio = False
         try:
             from cStringIO import StringIO
+            have_stringio = True
         except ImportError:
+            pass
+        try:    
             from StringIO import StringIO
+            have_stringio = True
+        except ImportError:
+            pass
+        try:
+            from io import StringIO
+            have_stringio = True
+        except ImportError:
+            pass
+        assert have_stringio and "no StringIO package available"
```

Comment:
would be cleaner to have the error string after a comma instead of abusing boolean operators...